Braze iOS v2 device mode updates (#969)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/braze.mdx

```
</span>
</li>
</ol>
      </TabPanel>
      <TabPanel>
        <div class="warningBlock">

>           This device mode integration is supported for Braze x.0.0 and above.

Author: @1abhishekpandey @pallabmaiti need inputs on the min. supported Braze version.

Reviewer: 4.4.4

```
</span>
</li>
</ol>
      </TabPanel>
      <TabPanel>
        <div class="warningBlock">

>           This device mode integration is supported for Braze 4.4.4 and above.

Reviewer 2: ```suggestion
          This device mode integration is supported for Braze v4.4.4 and above.
```

<span>

```ruby
pod 'RudderBraze', '~> 1.0.0'
```
</span>
</li>
<li>Run the <code class="inline-code">pod install</code> command.</li>
<li>Then, import the SDK depending on your preferred platform:
<span>

```swift
import RudderBraze
```
</span>
<span>

```objectivec
@import RudderBraze;
```
</span>
</li>
<li>Next, add the imports to your <code class="inline-code">AppDelegate</code> file under the <code class="inline-code">didFinishLaunchingWithOptions</code> method, as shown:
<br /><br />
<span>

```swift
let config: RSConfig = RSConfig(writeKey: WRITE_KEY)
            .dataPlaneURL(DATA_PLANE_URL)
RSClient.sharedInstance().configure(with: config)
RSClient.sharedInstance().addDestination(RudderBrazeDestination())
```
</span>
<span>

```objectivec
RSConfig *config = [[RSConfig alloc] initWithWriteKey:WRITE_KEY];
[config dataPlaneURL:DATA_PLANE_URL];
[[RSClient sharedInstance] configureWith:config];
[[RSClient sharedInstance] addDestination:[[RudderBrazeDestination alloc] init]];
```
</span>
</li>

> <li>To send the push notification events, refer to the <Link to="#sending-push-notification-events">Sending push notifications</Link> section below.

Reviewer 2: ```suggestion
<div class="infoBlock">
To send the push notification events, refer to the <Link to="#sending-push-notification-events">Sending push notifications</Link> section below.
</div>
```

IMO, it can be an infoblock instead of a step

Once you send a `group` event, RudderStack sends a custom attribute to Braze with the name as `ab_rudder_group_<groupId>` and the value as `true`. For example, if the `groupId` is `123456`, then RudderStack creates a custom attribute with the name `ab_rudder_group_123456` and sends it to Braze with its value to `true`.

## Sending push notification events

<div class="infoBlock">
This feature is applicable only for the <Link to="/sources/event-streams/sdks/rudderstack-ios-sdk/ios-v2/">RudderStack iOS SDK v2</Link>.
</div>

> For sending push notification events, add the following code to your <code class="inline-code">AppDelegate</code> file under the <code class="inline-code">didFinishLaunchingWithOptions</code> method:

Reviewer 2: ```suggestion
For sending the push notification events, add the following code to your <code class="inline-code">AppDelegate</code> file under the <code class="inline-code">didFinishLaunchingWithOptions</code> method:
```
